src/main/resources/application.yml

```diff
@@ -18,14 +18,13 @@ spring:
           issuer-uri: ${hmppsauth.baseurl}/issuer
           jwk-set-uri: ${hmppsauth.baseurl}/.well-known/jwks.json
   datasource:
-    driver-class-name: org.postgresql.Driver
     url: jdbc:postgresql://${postgres.uri}/${postgres.db}
     username: ${postgres.username}
     password: ${postgres.password}
   jpa:
-    database-platform: org.hibernate.dialect.PostgreSQL94Dialect
     hibernate:
-      ddl-auto: update
+      show_sql: true
```

Comment:
Can this be configurable? (We want to avoid this on kubernetes environments, AFAIK)

Reply:
this was left in by mistake. will remove. good spot.